### ISSUE TITLE
`:publishToMavenLocal` no longer skips `:signNebulaPublication`

### ISSUE THREAD

**Issue by timo-a** [moderneinc/rewrite-recipe-starter#52]
It seems like the last commit (15bdd28cfa504d275c528e8e56c43fa196599aff) broke existing functionality and `:publishToMavenLocal` no longer skipps `:signNebulaPublication`
Use case is I want to publish locally and have not set up any remote artifact repositories.

To reproduce, open the Readme in IntelliJ, click the green 'play' symbol, then 'execute line' for `./gradlew publishToMavenLocal`

c17f53fc3ef570047d772875e814676282e3d733 (commit before the last, everything works as expected)
````
22:14:12: Executing 'publishToMavenLocal'...


> Configure project :
Inferred project: rewrite-recipe-starter, version: 0.1.0-SNAPSHOT

> Task :compileJava
> Task :rewriteRecipeAuthorAttributionJava
> Task :rewriteRecipeAuthorAttributionResources UP-TO-DATE
> Task :copyAttribution
> Task :processResources UP-TO-DATE
> Task :classes
> Task :createPropertiesFileForJar UP-TO-DATE
> Task :writeManifestProperties
> Task :jar
> Task :javadoc
> Task :javadocJar UP-TO-DATE
> Task :sourcesJar
> Task :generateMetadataFileForNebulaPublication SKIPPED
> Task :generatePomFileForNebulaPublication
> Task :signNebulaPublication SKIPPED
> Task :publishNebulaPublicationToMavenLocal
> Task :publishToMavenLocal

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.7/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 12s
13 actionable tasks: 9 executed, 4 up-to-date
22:14:24: Execution finished 'publishToMavenLocal'.

````

15bdd28cfa504d275c528e8e56c43fa196599aff (last commit, does run `:signNebulaPublication`)
````
Execution failed for task ':signNebulaPublication'.
> Cannot perform signing task ':signNebulaPublication' because it has no configured signatory

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
For more on this, please refer to https://docs.gradle.org/8.7/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
BUILD FAILED in 23s
11 actionable tasks: 6 executed, 5 up-to-date

````

**Comment by timtebeek** [comment]
Thanks for pointing this out @timo-a ; that's a very basic requirement that's currently not working as hoped. It ties in with this matching change in https://github.com/openrewrite/rewrite-build-gradle-plugin/commit/7f0b11d93bf3eb0ebc4e300f41a4180584789c71 

I know Sam's out for the weekend; perhaps @pstreef has an idea how to resolve this one? I tried briefly but I'm not as familiar.

**Comment by timtebeek** [comment]
@sambsnyd any chance you could revisit this to allow Maven local repository installation from Gradle again?

**Comment by timtebeek** [comment]
Fixed in fad20338fb5bcab04818372f4846590dc8d55cd8
